packages/docusaurus-theme-classic/src/theme/Logo/index.tsx

```diff
@@ -32,7 +32,7 @@ export default function Logo(props: Props): JSX.Element {
       sources={sources}
       height={logo.height}
       width={logo.width}
-      alt={logo.alt || navbarTitle || title}
+      alt={logo.alt ?? (navbarTitle || title)}
```

Comment:
What do you think about the case of `alt: undefined` and `navbar: { title: "TypeScript ESLint" }`? Should we use `alt=""` as well (sounds more sensible to me)?

Reply:
I'm pretty much of the mindset that if any text appears in the same link as the logo, the logo should have `alt=""` - since at that point, the logo serves only to adorn that text.

Reply:
Sure - I'll pull that out into its own declaration though, so that way it's not all inlined.

Reply:
Updated